README.md

```diff
@@ -1 +1,154 @@
-# Portfolio-part-2-Microverse
\ No newline at end of file
+# Portfolio
+Portfolio: setup and mobile first
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
```

Comment:
- [ ] Kindly add this `section` in your `README` file, as of now I can see that there is no **Deployment** section or header which is a required header. Please keep in mind that except for the `FAQ` rest everything is important, so add it in the `README` to make your documentation professional not misleading.